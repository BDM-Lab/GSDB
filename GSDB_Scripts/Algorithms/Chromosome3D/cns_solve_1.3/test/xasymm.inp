remarks file: xasymm.inp 

evaluate ($low=76.0)
evaluate ($high=9.)
                                             
xray
   declare name=fobs type=complex domain=reciprocal end
   declare name=fpart domain=reciprocal type=complex end
   declare name=fcalc domain=reciprocal type=complex end
 
   A=185.1 B=185.1 C=185.1 ALPHA=90.0 BETA=90.0 GAMMA=90.0

   evaluate ($sg=I432) { space group No. 211 }
  symmetry=( X,Y,Z )
  symmetry=( -X,-Y,Z )
  symmetry=( -X,Y,-Z )
  symmetry=( X,-Y,-Z )
  symmetry=( Z,X,Y )
  symmetry=( Z,-X,-Y )
  symmetry=( -Z,-X,Y )
  symmetry=( -Z,X,-Y )
  symmetry=( Y,Z,X )
  symmetry=( -Y,Z,-X )
  symmetry=( Y,-Z,-X )
  symmetry=( -Y,-Z,X )
  symmetry=( Y,X,-Z )
  symmetry=( -Y,-X,-Z )
  symmetry=( Y,-X,Z )
  symmetry=( -Y,X,Z )
  symmetry=( X,Z,-Y )
  symmetry=( -X,Z,Y )
  symmetry=( -X,-Z,-Y )
  symmetry=( X,-Z,Y )
  symmetry=( Z,Y,-X )
  symmetry=( Z,-Y,X )
  symmetry=( -Z,Y,X )
  symmetry=( -Z,-Y,-X )
  symmetry=( 1/2+X,1/2+Y,1/2+Z )
  symmetry=( 1/2-X,1/2-Y,1/2+Z )
  symmetry=( 1/2-X,1/2+Y,1/2-Z )
  symmetry=( 1/2+X,1/2-Y,1/2-Z )
  symmetry=( 1/2+Z,1/2+X,1/2+Y )
  symmetry=( 1/2+Z,1/2-X,1/2-Y )
  symmetry=( 1/2-Z,1/2-X,1/2+Y )
  symmetry=( 1/2-Z,1/2+X,1/2-Y )
  symmetry=( 1/2+Y,1/2+Z,1/2+X )
  symmetry=( 1/2-Y,1/2+Z,1/2-X )
  symmetry=( 1/2+Y,1/2-Z,1/2-X )
  symmetry=( 1/2-Y,1/2-Z,1/2+X )
  symmetry=( 1/2+Y,1/2+X,1/2-Z )
  symmetry=( 1/2-Y,1/2-X,1/2-Z )
  symmetry=( 1/2+Y,1/2-X,1/2+Z )
  symmetry=( 1/2-Y,1/2+X,1/2+Z )
  symmetry=( 1/2+X,1/2+Z,1/2-Y )
  symmetry=( 1/2-X,1/2+Z,1/2+Y )
  symmetry=( 1/2-X,1/2-Z,1/2-Y )
  symmetry=( 1/2+X,1/2-Z,1/2+Y )
  symmetry=( 1/2+Z,1/2+Y,1/2-X )
  symmetry=( 1/2+Z,1/2-Y,1/2+X )
  symmetry=( 1/2-Z,1/2+Y,1/2+X )
  symmetry=( 1/2-Z,1/2-Y,1/2-X )
   asymm=( 0<=x <=1/2
     and  0<=y<=1/2
     and  0<=z<=1/4
     and  z<=min(x,1/2-x,y,1/2-y) )



   SCATter ( chemical C* ) 
      2.31000 20.8439 1.02000 10.2075 1.58860 .568700 .865000 51.6512 .215600
   SCATter ( chemical N* )
      12.2126 .005700 3.13220 9.89330 2.01250 28.9975 1.16630 .582600 -11.529
   SCATter ( chemical O* )
      3.04850 13.2771 2.28680 5.70110 1.54630 .323900 .867000 32.9089 .250800
   SCATter ( chemical S* )
      6.90530 1.46790 5.20340 22.2151 1.43790 .253600 1.58630 56.1720 .866900
   SCATter ( chemical P* )
      6.43450 1.90670 4.17910 27.1570 1.78000 0.52600 1.49080 68.1645 1.11490
   SCATter ( chemical FE* )
      11.1764 4.61470 7.38630 0.30050 3.39480 11.6729 0.07240 38.5566 0.97070

   SCATTER ( CHEMICAL XC* )
   15.6348 -.00740 7.95180 0.60890 8.43720 10.3116 0.85370 25.9905 -14.875

   nreflections=20000
 
   binresolution $low $high
   mapresolution $high
   tselection=( $high <= d <= $low and 10000. >= amplitude(fobs)>=0.1 ) 

   do (fcalc=10.) ( all )
    ?
   method=fft   lookup=false 
   fft memory=50000   end


   set message=all end
   declare name=map1  domain=real end
   do (map1=ft(fcalc)) ( all )

end


stop


