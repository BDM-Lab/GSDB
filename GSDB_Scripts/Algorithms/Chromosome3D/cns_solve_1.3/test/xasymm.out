          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:23 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks file: xasymm.inp 
 CNSsolve> 
 CNSsolve>evaluate ($low=76.0) 
 EVALUATE: symbol $LOW set to    76.0000     (real)
 CNSsolve>evaluate ($high=9.) 
 EVALUATE: symbol $HIGH set to    9.00000     (real)
 CNSsolve> 
 CNSsolve>xray 
 XRAY>   declare name=fobs type=complex domain=reciprocal end 
 XDECLARE: Object FOBS has been declared.
 XRAY>   declare name=fpart domain=reciprocal type=complex end 
 XDECLARE: Object FPART has been declared.
 XRAY>   declare name=fcalc domain=reciprocal type=complex end 
 XDECLARE: Object FCALC has been declared.
 XRAY> 
 XRAY>   A=185.1 B=185.1 C=185.1 ALPHA=90.0 BETA=90.0 GAMMA=90.0 
 XRAY> 
 XRAY>   evaluate ($sg=I432) { space group No. 211 } 
 Assuming literal string "I432"
 EVALUATE: symbol $SG set to "I432" (string)
 XRAY>  symmetry=( X,Y,Z ) 
 XRAY>  symmetry=( -X,-Y,Z ) 
 XRAY>  symmetry=( -X,Y,-Z ) 
 XRAY>  symmetry=( X,-Y,-Z ) 
 XRAY>  symmetry=( Z,X,Y ) 
 XRAY>  symmetry=( Z,-X,-Y ) 
 XRAY>  symmetry=( -Z,-X,Y ) 
 XRAY>  symmetry=( -Z,X,-Y ) 
 XRAY>  symmetry=( Y,Z,X ) 
 XRAY>  symmetry=( -Y,Z,-X ) 
 XRAY>  symmetry=( Y,-Z,-X ) 
 XRAY>  symmetry=( -Y,-Z,X ) 
 XRAY>  symmetry=( Y,X,-Z ) 
 XRAY>  symmetry=( -Y,-X,-Z ) 
 XRAY>  symmetry=( Y,-X,Z ) 
 XRAY>  symmetry=( -Y,X,Z ) 
 XRAY>  symmetry=( X,Z,-Y ) 
 XRAY>  symmetry=( -X,Z,Y ) 
 XRAY>  symmetry=( -X,-Z,-Y ) 
 XRAY>  symmetry=( X,-Z,Y ) 
 XRAY>  symmetry=( Z,Y,-X ) 
 XRAY>  symmetry=( Z,-Y,X ) 
 XRAY>  symmetry=( -Z,Y,X ) 
 XRAY>  symmetry=( -Z,-Y,-X ) 
 XRAY>  symmetry=( 1/2+X,1/2+Y,1/2+Z ) 
 XRAY>  symmetry=( 1/2-X,1/2-Y,1/2+Z ) 
 XRAY>  symmetry=( 1/2-X,1/2+Y,1/2-Z ) 
 XRAY>  symmetry=( 1/2+X,1/2-Y,1/2-Z ) 
 XRAY>  symmetry=( 1/2+Z,1/2+X,1/2+Y ) 
 XRAY>  symmetry=( 1/2+Z,1/2-X,1/2-Y ) 
 XRAY>  symmetry=( 1/2-Z,1/2-X,1/2+Y ) 
 XRAY>  symmetry=( 1/2-Z,1/2+X,1/2-Y ) 
 XRAY>  symmetry=( 1/2+Y,1/2+Z,1/2+X ) 
 XRAY>  symmetry=( 1/2-Y,1/2+Z,1/2-X ) 
 XRAY>  symmetry=( 1/2+Y,1/2-Z,1/2-X ) 
 XRAY>  symmetry=( 1/2-Y,1/2-Z,1/2+X ) 
 XRAY>  symmetry=( 1/2+Y,1/2+X,1/2-Z ) 
 XRAY>  symmetry=( 1/2-Y,1/2-X,1/2-Z ) 
 XRAY>  symmetry=( 1/2+Y,1/2-X,1/2+Z ) 
 XRAY>  symmetry=( 1/2-Y,1/2+X,1/2+Z ) 
 XRAY>  symmetry=( 1/2+X,1/2+Z,1/2-Y ) 
 XRAY>  symmetry=( 1/2-X,1/2+Z,1/2+Y ) 
 XRAY>  symmetry=( 1/2-X,1/2-Z,1/2-Y ) 
 XRAY>  symmetry=( 1/2+X,1/2-Z,1/2+Y ) 
 XRAY>  symmetry=( 1/2+Z,1/2+Y,1/2-X ) 
 XRAY>  symmetry=( 1/2+Z,1/2-Y,1/2+X ) 
 XRAY>  symmetry=( 1/2-Z,1/2+Y,1/2+X ) 
 XRAY>  symmetry=( 1/2-Z,1/2-Y,1/2-X ) 
 XRAY>   asymm=( 0<=x <=1/2 
 ASYMmetric>     and  0<=y<=1/2 
 ASYMmetric>     and  0<=z<=1/4 
 ASYMmetric>     and  z<=min(x,1/2-x,y,1/2-y) ) 
 XRAY> 
 XRAY> 
 XRAY> 
 XRAY>   SCATter ( chemical C* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      2.31000 20.8439 1.02000 10.2075 1.58860 .568700 .865000 51.6512 .215600 
 XRAY>   SCATter ( chemical N* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      12.2126 .005700 3.13220 9.89330 2.01250 28.9975 1.16630 .582600 -11.529 
 XRAY>   SCATter ( chemical O* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      3.04850 13.2771 2.28680 5.70110 1.54630 .323900 .867000 32.9089 .250800 
 XRAY>   SCATter ( chemical S* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      6.90530 1.46790 5.20340 22.2151 1.43790 .253600 1.58630 56.1720 .866900 
 XRAY>   SCATter ( chemical P* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      6.43450 1.90670 4.17910 27.1570 1.78000 0.52600 1.49080 68.1645 1.11490 
 XRAY>   SCATter ( chemical FE* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=      11.1764 4.61470 7.38630 0.30050 3.39480 11.6729 0.07240 38.5566 0.97070 
 XRAY> 
 XRAY>   SCATTER ( CHEMICAL XC* ) 
 SELRPN:      0 atoms have been selected out of      0
 SCATTER_A1=   15.6348 -.00740 7.95180 0.60890 8.43720 10.3116 0.85370 25.9905 -14.875 
 XRAY> 
 XRAY>   nreflections=20000 
 XRAY: allocating space for   20000 reflections.
 XRAY> 
 XRAY>   binresolution $low $high 
 XRAY>   mapresolution $high 
 XRAY>   tselection=( $high <= d <= $low and 10000. >= amplitude(fobs)>=0.1 ) 
 XRAY> 
 XRAY>   do (fcalc=10.) ( all ) 
 Total of         0 structure factor elements were selected.
 XRAY>    ? 
 ---------------------------diffraction-data-----------------------------------
 | a=185.10, b=185.10, c=185.10, alpha= 90.00, beta= 90.00, gamma= 90.00
 | ANOMalous=FALSe
 | Number or unique reflections read=     0
 | resolution range covered by these reflections:       0.00 to       0.00
 | Hmax=    0 Kmax=    0 Lmax=    0 Hmin=    0 Kmin=    0 Lmin=    0
 ------------------------------------------------------------------------------
 XRAY>   method=fft   lookup=false 
 XRAY>   fft memory=50000   end 
 XRAY> 
 XRAY> 
 XRAY>   set message=all end 
 XRAY>   declare name=map1  domain=real end 
 XRAY>   do (map1=ft(fcalc)) ( all ) 
 DO: bypassing selection.
 DO: executing operations right of FT.
 XSFAL: allocating space for complex reciprocal space object.
 Total of         0 structure factor elements were selected.
 XMAPASU: using grid [  64,  64,  64] and sublattice [  32,  32,  32]
 Minimum brick that covers asymmetric unit:
   A=     0,...,    32  B=     0,...,    32  C=     0,...,    16
 Number of deleted points in ASU=        1009
 Number of non-zero lattice points in ASU=        5536
 Number of special positions in ASU=         126
 XMAPAL: allocating space for real space object.
 DO: bypassing operations left of FT.
 XRAY> 
 XRAY>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:    10451088 bytes
           Maximum dynamic memory overhead:           432 bytes
           Program started at: 16:08:23 on 05-Jul-2018
           Program stopped at: 16:08:23 on 05-Jul-2018
           CPU time used:       0.4675 seconds
          ============================================================
